cluster/gce/gci/master.yaml

```diff
@@ -74,6 +74,33 @@ write_files:
       [Install]
       WantedBy=kubernetes.target
 
+  - path: /etc/systemd/system/kube-logrotate.timer
+    permissions: 0644
+    owner: root
+    content: |
+      [Unit]
+      Description=Hourly kube-logrotate invocation
+
+      [Timer]
+      OnCalendar=hourly
+
+      [Install]
+      WantedBy=kubernetes.target
+
+  - path: /etc/systemd/system/kube-logrotate.service
```

Comment:
Yes. As mentioned on https://www.freedesktop.org/software/systemd/man/systemd.timer.html, "a timer file foo.timer activates a matching service foo.service". So in this case, kube-logrotate.timer activates a kube-logrotate.service. I have explained this in the PR description now. PTAL.